charts/gateway/values.yaml

```diff
@@ -31,6 +31,9 @@ imagePullSecrets: []
 envFrom: []
 
 config:
+  routingRules:
+    rulesEngineEnabled: false
+    rulesConfigPath: "/opt/gateway/routing_rules/routing_rules.yml"
```

Comment:
This change appears to be incomplete. The routing-rules configmap is not mounted anywhere